src/components/CardGroup.tsx

```diff
@@ -24,7 +27,9 @@ export const CardHeader = (props: CardHeaderProps) => {
       onPress={onPress}
       style={[styles.headerContainer, { borderBottomColor: isChoosen ? palette.textSecondary : palette.textGrey }]}
     >
-      <Text style={styles.headerTitle}>{title}</Text>
+      <Text style={styles.headerTitle}>
+        {title === CONST.receive ? i18n.filterTransactions.received : i18n.filterTransactions.sent}
```

Comment:
if you hardcode this here than CardGroup is not a reusable component no more, that it was I think it's better to put this condition in the  `onCardPressAction={title =>  this.props.updateTransactionType(i18n.filterTransactions.received ? CONST.receive :  CONST.send)}`

Reply:
Yes, but we don't use CardGroup anywhere else and this way you put translating strings as a key which is not cool.

Reply:
I don't see the situation when it won't work with translation string but I can imagine the situation when we will have to use in the future same component and the person who will have to that will be confused what is the purpose of this condition if it won't know the context of the whole app. Also, that person will have to understand the logic behind TransactionList to make this component reusable again